app/Console/Commands/SendEmailNotVerificationUser.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Mail\EmailConfirm;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Mail;
+
+class SendEmailNotVerificationUser extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'user:notify-verification';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send email not verified user';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $users = User::all()->whereNull('email_verified_at');
```

Comment:
тут тоже самое, метод all достает из базы все записи
а фильтрация происходит уже в php
лучше сразу в базе отфильтровать
```suggestion
        $users = User::query()->whereNull('email_verified_at')->all();
```